code/modules/antagonists/fugitive/fugitive_outfits.dm

```diff
@@ -112,3 +112,33 @@
 	W.assignment = "Police Officer"
 	W.registered_name = H.real_name
 	W.update_label()
+
+/datum/outfit/gadget
+	name = "Inspector Gadget"
+	uniform = /obj/item/clothing/under/color/grey
+	suit = /obj/item/clothing/suit/det_suit/grey
+	head = /obj/item/clothing/head/fedora
+	shoes = /obj/item/clothing/shoes/sneakers/brown
+	ears = /obj/item/radio/headset
+	implants = list(/obj/item/implant/explosive, /obj/item/organ/cyberimp/arm/toolset)
```

Comment:
Aw hell naw, this is going to make the hunter's job miserable, if they kill him, his body disappers, if they dont kill him, he will blow up on you